examples/kubernetes/addons/etcd-operator/cilium-deployment.yaml

```diff
@@ -98,7 +98,7 @@ spec:
                   optional: true
                   key: clean-cilium-state
       containers:
-        - image: docker.io/cilium/cilium:v1.2.1
+        - image: docker.io/cilium/cilium:v1.2.2
```

Comment:
Let's leave this with 1.2.0 until we have etcd-operator CI tests merged

Reply:
Why is 1.2.1 better than 1.2.2? I would understand 1.2.0